app/src/main/java/org/mozilla/rocket/content/games/GamesViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package org.mozilla.rocket.content.games
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.mozilla.rocket.adapter.DelegateAdapter
+
+class GamesViewModel : ViewModel() {
+
+    val browserGamesState = MutableLiveData<State>().apply { value = State.Idle }
```

Comment:
Just a thought, we could combine these two livedata into one, like: MutableLiveData<State<List<DelegateAdapter.UIModel>>>()